### PR TITLE
Show download count info in release list

### LINKED ISSUE #9265: Show the count of file downloads in Release page.
Show downloads here.    
image

### PR DESCRIPTION
**Pull request by lafriks**
Fixes #9265

image

### CODE REVIEW

**Comment by jolheiser** [comment]
How does this compare to https://github.com/dustin/go-humanize/blob/master/comma.go#L11-L44?

It doesn't need to handle negative numbers in the context of this PR (https://play.golang.org/p/5wF98ePRPc-) but perhaps it would be nice to have so we can re-use this function in other areas?

**Comment by lafriks** [comment]
I did not want to add new library just for this but it could probably be useful also in other places

**Comment by jolheiser** [comment]
Yes, agreed we don't need that whole library. Maybe just extending it to negatives, if possible?  

iirc there is an issue for humanizing diff counts, which is the only thing I can think of with negatives but I'm sure there are more.

**Comment by lafriks** [comment]
Actually it is second function already so I just added go-humanize (`FileSize` was actually copied over  already)

File: modules/base/tool.go [hunk 1]
```diff
@@ -214,40 +214,15 @@ func AvatarLink(email string) string {
...
-
 // FileSize calculates the file size and generate user-friendly string.
 func FileSize(s int64) string {
-	sizes := []string{"B", "KB", "MB", "GB", "TB", "PB", "EB"}
-	return humanateBytes(uint64(s), 1024, sizes)
+	return humanize.Bytes(uint64(s))
+}
+
+// PrettyNumber produces a string form of the given number in base 10 with
+// commas after every three orders of magnitud
```
Comment (jolheiser) [inline modules/base/tool.go:223]:
```suggestion
// commas after every three orders of magnitude
```

File: modules/templates/helper.go [hunk 2]
```diff
@@ -93,6 +93,7 @@ func NewFuncMap() []template.FuncMap {
 		"TimeSinceUnix": timeutil.TimeSinceUnix,
 		"RawTimeSince":  timeutil.RawTimeSince,
 		"FileSize":      base.FileSize,
+		"PrettyNumber":  base.PrettyNumber,
```
Comment (jolheiser) [inline modules/templates/helper.go:96]: Any reason to go through this func when all it does is wrap the humanize func?  
Same with `FileSize` now that it uses humanize.

**Review by jolheiser — APPROVED** [review]
A nit and question, otherwise LGTM

File: modules/templates/helper.go (hunk 2 above)
Comment (lafriks) [reply modules/templates/helper.go:96]: Because `base.FileSize` is used all over the code and to have more flexibility just in case

**Review by 6543 — APPROVED** [review]
nice - you only have to edit the tests

**Pull request merged**